### PR TITLE
Missing return types in method signatures

### PR DESCRIPTION
**Pull request by dorlinski**
[Problem] Some method does not have return value in their declaration despite that they return value
[Solution] Fill missing return types

Signed-off-by: Dominik Orliński <d.orlinski@samsung.com>

### CODE REVIEW

File: docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Radio.htm [hunk 1]
```diff
@@ -73,14 +73,14 @@ <h3>Summary</h3>
...
-			<pre class="intable prettyprint"><a href="#method-enable">enable</a>() </pre>
+			<pre class="intable prettyprint"><a href="#method-enable">Radio enable</a>() </pre>
 		</td>
 		<td><p>Enables the component.</p></td>
 	</tr>
 
 	<tr>
 		<td>
-			<pre class="intable prettyprint"><a href="#method-value">value</a>() </pre>
+			<pre class="intable prettyprint"><a href="#method-value">{string | number}value</a>() </pre>
```
Comment (mijin85cho) [inline docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Radio.htm]: Don't we need a space between "{string | number}" and "value"?

Comment (dorlinski) [reply docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Radio.htm]: Fixed

File: docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_SectionChanger.htm [hunk 2]
```diff
@@ -198,7 +198,7 @@ <h3>Summary</h3>
 				</tr>
 				<tr>
 					<td>
-						<pre class="intable prettyprint"><a href="#method-getActiveSectionIndex">getActiveSectionIndex</a>() </pre>
+						<pre class="intable prettyprint"><a href="#method-getActiveSectionIndex">number getActiveSectionIndex</a>() </pre>
```
Comment (mijin85cho) [inline docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_SectionChanger.htm]: Just a question. 
Is it OK to use "number", not "Number"?

In other pages, the return types start with capital letter, while in some pages start with small letter.

File: docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_sectionchanger.htm [hunk 3]
```diff
@@ -203,7 +203,7 @@ <h3>Summary</h3>
 				</tr>
 				<tr>
 					<td>
-						<pre class="intable prettyprint"><a href="#method-getActiveSectionIndex">getActiveSectionIndex</a>() </pre>
+						<pre class="intable prettyprint"><a href="#method-getActiveSectionIndex">number getActiveSectionIndex</a>() </pre>
```
Comment (mijin85cho) [inline docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_sectionchanger.htm]: Just a question.
Is it OK to use "number", not "Number"?

File: docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_SectionChanger.htm (hunk 2 above)
Comment (dorlinski) [reply docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_SectionChanger.htm]: I just rewrite this type from "return value" table, so I left it as it is to keep consistency of naming in this file.

Comment (mijin85cho) [reply docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_SectionChanger.htm]: Thanks for the answer @dorlinski.

@TomaszLukawskiSam If you think it is good enough to merge this PR, would you please re-approve this PR?

File: docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_sectionchanger.htm (hunk 3 above)
Comment (TomaszLukawskiSam) [reply docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_sectionchanger.htm]: > Just a question.
> Is it OK to use "number", not "Number"?

Properly lower case only for simple types:
- number
- string

Rest of types should be from capital letter like:
- Object
- Event
- HTMLElement

Comment (TomaszLukawskiSam) [reply docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_sectionchanger.htm]: @dorlinski 
Would you change "Number" to "number"? Thank you.

Comment (dorlinski) [reply docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_sectionchanger.htm]: Yes, but maybe I will do it in separate PR, in which I correct all?

Comment (dorlinski) [reply docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_sectionchanger.htm]: Do I have to change all types to lowercase (Boolean -> boolean, Number -> number, String -> string)? And maybe it better to use upper case everywhere?

Comment (dorlinski) [reply docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_sectionchanger.htm]: Done

**Comment by mijin85cho** [comment]
FYI. The Web API Reference files for 5.0, the latest version including this PR(#653) on the master branch was synced to the Developer Tizen Site.

**Pull request merged**